here-naksha-app-service/src/main/java/com/here/naksha/app/service/http/tasks/WriteFeatureApiTask.java

```diff
@@ -18,12 +18,9 @@
  */
 package com.here.naksha.app.service.http.tasks;
 
-import static com.here.naksha.app.service.http.apis.ApiParams.ADD_TAGS;
-import static com.here.naksha.app.service.http.apis.ApiParams.PREFIX_ID;
-import static com.here.naksha.app.service.http.apis.ApiParams.REMOVE_TAGS;
-import static com.here.naksha.app.service.http.apis.ApiParams.SPACE_ID;
-import static com.here.naksha.app.service.http.apis.ApiParams.pathParam;
+import static com.here.naksha.app.service.http.apis.ApiParams.*;
```

Comment:
nth: favor specific imports over greedy ones

Reply:
ok I adjusted it